doc/lua_api.md

```diff
@@ -291,6 +291,18 @@ in one of its parents, the parent's file is used.
 Although it is discouraged, a mod can overwrite a media file of any mod that it
 depends on by supplying a file with an equal name.
 
+Of the model file formats, only a subset is supported:
+
+Simple textured meshes (with multiple textures), optionally with normals.
+The .x and .b3d formats additionally support skeletal animation.
+
+Do not rely on glTF features not being supported; they may be supported in the future.
```

Comment:
Modders might wonder *what* is yet not supported.

Reply:
Everything that's not part of what's mentioned above, for example the extensive glTF materials API, or embedded images, or (in this PR) skeletal animations. This PR just makes glTF an alternative to other static model formats used in Minetest, no more and no less. It unlocks no new features yet, that is for later PRs.

It is less feasible to try to list in full what we don't support than to just state briefly what we do support (as is done above, though we should probably be more precise).

This is basically a disclaimer to make sure we can just add support for these features later without modders complaining that suddenly their model glows (because the material properties, which used to be ignored, were wrong), etc.

Reply:
Improved the docs in https://github.com/minetest/minetest/pull/14557/commits/0b03dd4fa957aff49147b56675d14ff705c07a75, including a rough high-level overview of what isn't supported.